docs/Configuration.adoc

```diff
@@ -67,5 +67,5 @@ url `${GITHUB_REPO_GIT_URL}`, branchspec `${GITHUB_BRANCH_NAME}`.
 - `GITHUB_BRANCH_CAUSE_SKIP` - `true` or `false`
 
 ===== Specific for repo:
-- `GITHUB_REPO_GIT_URL` - `git://github.com/KostyaSha-org/test-branches.git`
```

Comment:
was it changed in github api? https://github.com/KostyaSha/github-integration-plugin/blob/afbb8ca848c2e5967136afe6f499f012633321ac/github-pullrequest-plugin/src/main/java/com/github/kostyasha/github/integration/generic/GitHubRepoEnv.java#L12

Reply:
I can't `git clone git://github.com/jenkinsci/git-client-plugin.git` with either command line git or JGit.  I don't know if it was changed in github api, but as far as I can tell that URL is no longer a viable way to access a GitHub repository.

Reply:
If i correctly understand this protocol doesn't work: 1) for public repos 2) for old key types. Variable name is git and it's for git protocol. We can add one more variable and call it `HTTPS_URL` just changing a doc here wouldn't influence :)

Reply:
I think you've mixed two different things in your reading of the GitHub blog post.  The blog post announced two distinct changes:

* End of support for `git://` protocol access to GitHub repositories
* End of support for `ssh://` protocol access to GitHub repositories using outdated key types

The `git://` protocol doesn't work for any GitHub repository, public or private.  Before the change announced in the GitHub blog post, it only worked for public repositories.

The `git://` protocol is [unauthenticated and unencrypted](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_the_git_protocol).  The `https://` protocol is encrypted and can optionally be authenticated.

The GitHub change to drop old key types applies to the `ssh://` protocol, not to the `git://` protocol.